contracts/deploy/sonic/002_oracle_router.js

```diff
@@ -0,0 +1,32 @@
+const { deployOnSonic } = require("../../utils/deploy-l2.js");
+const { deployOracles } = require("../deployActions");
+
+module.exports = deployOnSonic(
+  {
+    deployName: "002_oracle_router",
+    forceSkip: false,
+  },
```

Comment:
Quick question to learn a bit more, is there a reason not to set `reduceQueueTime` to true here?

Reply:
Currently the timelock is already configured to have 60s delay time on sonic chain. No need to reduce it